### PR TITLE
Add a flag to show/hide code

### PR DESCRIPTION
**Pull request by Arinerron**

### CODE REVIEW

File: generate.py [hunk 1]
```diff
@@ -25,9 +25,12 @@
 
 EXAMPLE_TEMPLATE = '''<br>    {example}'''
 
-CODE_TEMPLATE = '''
+CODE_TEMPLATE = '''            <hr>
+            <details>
+                <summary>code</summary>
```
Comment (mysterywave) [inline generate.py:30]: oh cool you can comment

File: generate.py [hunk 2]
```diff
@@ -25,9 +25,12 @@
...
                 <pre class="textblock">{defines}</pre>
-                <pre class="textblock">{instructions}</pre>'''
```
Comment (mysterywave) [inline generate.py:30]: nice code

File: generate.py [hunk 3]
```diff
@@ -115,7 +119,7 @@ def generate_code(data):
     
     output = formatter({
         'tables' : generate_tables(data),
-        'code' : generate_code(data)
```
Comment (mysterywave) [inline generate.py:118]: these two lines are pretty nice too

File: template.html [hunk 4]
```diff
@@ -96,11 +96,7 @@
         <div id="maindiv">
             <h1 id="welcome">SALL Docs</h1>
 {tables}
-            <hr>
-            <details>
-                <summary>code</summary>
 {code}
-            </details>
```
Comment (mysterywave) [inline template.html:103]: pretty good

File: generate.py (hunk 3 above)
Comment (Arinerron) [reply generate.py:118]: <3

File: generate.py (hunk 1 above)
Comment (Arinerron) [reply generate.py:30]: yeah lol

**Pull request merged**